canonical/ubuntu-advantage-client
# gcp: add pro license for Jammy

Now we have a PRO license ID for Jammy, and the license-check should run there as well.

## Test Steps
Run the integration test to make sure the job is running.
Output from a local run:
```
@wip @series.lts @uses.config.contract_token @uses.config.machine_type.gcp.generic
  Scenario Outline: license_check job should run periodically on gcp generic lts -- @1.4 version  # features/license_check.feature:42
    Given a `jammy` machine with ubuntu-advantage-tools installed                                 # features/steps/steps.py:55
    Then I verify the `ua-license-check` systemd timer is scheduled to run within `10` minutes    # features/steps/steps.py:766
    When I run `systemctl start ua-license-check.service` with sudo                               # features/steps/steps.py:266
    When I wait `5` seconds                                                                       # features/steps/steps.py:474
    Then I verify the `ua-license-check` systemd timer is scheduled to run within `10` minutes    # features/steps/steps.py:766
    When I wait `5` seconds                                                                       # features/steps/steps.py:474
    When I attach `contract_token` with sudo                                                      # features/steps/steps.py:405
    Then I verify the `ua-license-check` systemd timer is disabled                                # features/steps/steps.py:731
    When I run `ua detach --assume-yes` with sudo                                                 # features/steps/steps.py:266
    And I wait `5` seconds                                                                        # features/steps/steps.py:474
    Then I verify the `ua-license-check` systemd timer is scheduled to run within `10` minutes    # features/steps/steps.py:766
    When I run `systemctl stop ua-license-check.timer` with sudo                                  # features/steps/steps.py:266
    When I run `rm /var/lib/ubuntu-advantage/marker-license-check` with sudo                      # features/steps/steps.py:266
    When I run `systemctl stop ua-license-check.timer` with sudo                                  # features/steps/steps.py:266
    Then I verify the `ua-license-check` systemd timer is disabled                                # features/steps/steps.py:731
    When I run `touch /var/lib/ubuntu-advantage/marker-license-check` with sudo                   # features/steps/steps.py:266
    Then I verify the `ua-license-check` systemd timer is scheduled to run within `10` minutes    # features/steps/steps.py:766

1 feature passed, 0 failed, 19 skipped
1 scenario passed, 0 failed, 324 skipped
17 steps passed, 0 failed, 5154 skipped, 0 undefined
Took 7m27.028s
```

## Desired commit type

## Checklist
